app/src/main/res/layout/completed_experience.xml

```diff
@@ -1,52 +1,64 @@
+    android:layout_height="wrap_content">
 
     <ImageView
+        android:id="@+id/completed_experience_logo"
         android:layout_width="80dp"
         android:layout_height="80dp"
-        android:id="@+id/completed_experience_logo"
-        android:scaleY="2.5"
-        android:scaleX="2.5"
-        android:contentDescription="@string/completed_experience_logo_description"
         android:layout_marginHorizontal="16dp"
         android:layout_marginVertical="8dp"
+        android:contentDescription="@string/completed_experience_logo_description"
+        android:scaleX="2.5"
+        android:scaleY="2.5"
+        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent" />
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
+        android:id="@+id/completed_experience_title"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:id="@+id/completed_experience_title"
         android:layout_marginStart="16dp"
+        android:layout_marginTop="8dp"
         android:layout_marginEnd="16dp"
-        android:layout_marginTop="16dp"
-        android:layout_marginBottom="4dp"
         android:textAppearance="?attr/textAppearanceSubtitle1"
         android:textColor="?attr/colorOnSurface"
-        android:alpha="0.87"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintStart_toEndOf="@+id/completed_experience_logo"
+        app:fontFamily="@font/pt_sans_bold"
+        app:layout_constraintBottom_toTopOf="@+id/completed_experience_description"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintBottom_toTopOf="@+id/completed_experience_description" />
+        app:layout_constraintStart_toEndOf="@+id/completed_experience_logo"
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
+        android:id="@+id/completed_experience_description"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:id="@+id/completed_experience_description"
         android:layout_marginStart="16dp"
         android:layout_marginEnd="16dp"
-        android:layout_marginBottom="16dp"
         android:textAppearance="?attr/textAppearanceBody2"
         android:textColor="?attr/colorOnSurface"
-        android:alpha="0.87"
-        app:layout_constraintStart_toEndOf="@+id/completed_experience_logo"
+        app:fontFamily="@font/pt_sans"
+        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/completed_experience_title"
-        app:layout_constraintBottom_toBottomOf="parent" />
+        app:layout_constraintStart_toEndOf="@+id/completed_experience_logo"
+        app:layout_constraintTop_toTopOf="parent" />
```

Comment:
This constraint should be set to the above `TextView`

Reply:
I did in that way to center vertically the image and the description without dealing with margins and so on.

Reply:
The image is centered anyway by its own constraints. What you're looking for is `app:layout_constraintVertical_chainStyle="packed"`. Check out the latest corrections I pushed.